tests/testsuite/serviceaccount.go

```diff
@@ -50,6 +51,9 @@ func createServiceAccounts() {
 	createServiceAccount(ViewServiceAccountName)
 	createRoleBinding(ViewServiceAccountName, "kubevirt.io:view")
 
+	createServiceAccount(ViewInstancetypeServiceAccountName)
+	createClusterRoleBinding(ViewInstancetypeServiceAccountName, "instancetype.kubevirt.io:view")
```

Comment:
```suggestion
	createClusterRoleBinding(fmt.Sprintf("%s-%d", ViewInstancetypeServiceAccountName, GinkgoParallelProcess()), "instancetype.kubevirt.io:view")
```

When running tests in parallel you need to give the created `ClusterRoleBindings` individual names, as they are on the cluster level but refer to ServiceAccounts in individual namespaces. This should solve the failing tests. Note: Cleanup needs also to be fixed.